### DISCUSSION TITLE
Advantage/reasons of choosing backend for torch.compile: openvino vs. openvino_ts backend

### DISCUSSION THREAD

**Discussion by js333031** [openvinotoolkit/openvino#23943] [Q&A] [answered]
Per the link below, please explain when to select openvino vs. openvino_ts backend.

[https://docs.openvino.ai/2024/openvino-workflow/torch-compile.html](https://docs.openvino.ai/2024/openvino-workflow/torch-compile.html)

**Comment by siddhant-0707** [comment, marked as answer]
Hey @js333031,

`torch.compile(model, backend='openvino')` tends to be faster in all cases. Moreover `openvino_ts` will probably be removed in upcoming releases. You can learn why `torch.compile` which is equivalent to `backend='openvino'` is faster than torchscript [here](https://pytorch.org/tutorials/intermediate/torch_compile_tutorial.html).
